packages/template/base/tmpl/_appveyor.yml

```diff
@@ -8,7 +8,7 @@ cache:
   - node_modules
 branches:
   only:
-    - master
+    - main
```

Comment:
Bringing over a comment from @malept in a previous PR - we should probably just delete this file before stable release. For the purposes of this PR, we can leave the change in, but we should open a separate PR removing the file and other unused template files.